core/ViewDataTable/Config.php

```diff
@@ -463,6 +463,14 @@ class Config
     public $export_limit = '';
 
     /**
+     * Message to show if not data is available for the report
+     * Defaults to `CoreHome_ThereIsNoDataForThisReport` if not set
+     *
+     * @var string
+     */
```

Comment:
Added in https://github.com/piwik/piwik/commit/6af9851af0a91196361c8e392a067b22b0f6113c